### PR TITLE
B3dm legacy check in tools + cleanup

### PR DESCRIPTION
**Pull request by lilleyse**
For spec changes: https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/200

### CODE REVIEW

**Comment by pjcozzi** [comment]
@shehzan10 do you want to look at this when you have time?

File: tools/specs/data/TilesetOfTilesets/tileset3/tileset3.json [hunk 1]
```diff
@@ -20,4 +20,4 @@
       "url": "ll.b3dm"
     }
   }
-}
```
Comment (shehzan10) [inline tools/specs/data/TilesetOfTilesets/tileset3/tileset3.json]: Do you want to undo this change? There are no other changes in this file.

Comment (lilleyse) [inline tools/specs/data/TilesetOfTilesets/tileset3/tileset3.json]: Not sure how that got in... I reverted it though.

File: tools/lib/glbToI3dm.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 'use strict';
-
 var Cesium = require('cesium');
+var getBufferPadded = require('./getBufferPadded');
+var getJsonBufferPadded = require('./getJsonBufferPadded');
+
 var defaultValue = Cesium.defaultValue;
```
Comment (shehzan10) [inline tools/lib/glbToI3dm.js]: Just ran jsHint, `defaultValue` is no longer used.
